### DISCUSSION TITLE
FAQ: Cannot login (Cannot create Apple ID Error)

### DISCUSSION THREAD

**Discussion by cryptofyre** [ciderapp/Cider#1010] [Announcements]
## Issue
The issue mentioned in the title has been an extremely common one recently with newcomers. I remind you again that you need an **ACTIVE** Apple Music subscription that is preferably not a trial as we haven’t tested those.

As for context of the error, we don’t really know. Apple has some illusive error’s and shit sometimes that we just don’t have control over at this moment in time.

## tl;dr
Make sure you have an **ACTIVE** Apple Music subscription and Apple ID in good standing (aka. not locked, or restricted.) also **avoid** business firewall’s and VPN’s when possible as Apple has been a little picky on those kind of network situations.

**Comment by GamingLiamStudios** [comment]
I would just like to say that I rocked a trial AM subscription for 3 months using Cider with no problem.

**Comment by TechStudent10** [comment]
Just a quick question, is the requirement for a subscription a thing on Apple's side, or did you add this requirement intentionally? Because all I want is a good way to listen to Apple Podcasts on my Windows PC, and I don't need a subscription to listen to podcasts.

**Reply by cryptofyre** [reply]
This is on Apple's side. MusicKit includes the tools to use the Podcasts features which happen to be bundled into the mess with AM subscriptions. We'll be digging into this further with the new podcasts app we're working on called Fritter, more updates to come.

**Reply by TechStudent10** [reply]
That sounds cool. Is there a repo for Fritter or is it still in very early development?
